pkg/operator/ceph/config/monstore.go

```diff
@@ -93,7 +93,7 @@ func (m *MonStore) SetIfChanged(who, option, value string) (bool, error) {
 // Set sets a config in the centralized mon configuration database.
 // https://docs.ceph.com/docs/master/rados/configuration/ceph-conf/#monitor-configuration-database
 func (m *MonStore) Set(who, option, value string) error {
-	logger.Infof("setting %q=%q=%q option to the mon configuration database", who, option, value)
```

Comment:
Can we continue logging the full information using `logger.Trace()` here? Sometimes, these logs are needed for debugging code issues in calling reconciliations. Trace level logs are hidden under `TRACE_INSECUCRE` log level in rook, so very few users should/will be using it in the real world.

Reply:
@BlaineEXE is it makes sense to merge updated `Info` statements and dicuss adding trace level logs on Monday to add them in a separate PR?

Reply:
Separate PR seems fine to me. Fixing info leak is definitely higher priority than making sure we have access to rarely-needed (but useful) debug info.